app/App.tsx

```diff
@@ -58,11 +58,16 @@ export const App = () => {
   const [hamburgerOpen, setHamburgerOpen] = useState(false);
   const [popUpMessage, setPopUpMessage] = useState<PopupMessageProp>({ message: '', visible: false, type: 'success' });
   const [userLocation, setUserLocation] = useState<GeoCoordinates>(COORDS_MID_SAN_FRANCISCO);
+  const [userLocationPromiseReturned, setUserLocationPromiseReturned] = useState(false);
```

Comment:
I'm unsure if this is the best/most React-y way to do this – I'm using this AppContext boolean property to determine whether we can initialize the map (and thus set the center to the user's actual coords or SF default coords) to avoid the existing map flash bug that will become more apparent as our site becomes more "map-centric". Thoughts or suggestions are welcome